src/components/subreddits/index.js

```diff
@@ -3,14 +3,30 @@ import React from 'react';
 import Subreddit from './item';
 
 export default class SubredditsContainer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      selected: null
```

Comment:
since this is supposed to be a string, it will be good to just set it as '' as a default